planter/planter.sh

```diff
@@ -19,7 +19,7 @@
 set -o errexit
 set -o nounset
 IMAGE_NAME="gcr.io/k8s-testimages/planter"
-TAG="${TAG:-0.7.0-1}"
```

Comment:
if we're dropping the suffixes, we probably should have a `docker pull` in this script then?

Reply:
Yes, I thought I'd already patched that in, I've been fixing that right now
/hold

Reply:
done
/hold cancel